packages/sdk-react/src/mocks/reminders/remindersHandlers.ts

```diff
@@ -0,0 +1,174 @@
+  >('*/payment_reminders', async ({ request }) => {
+    const jsonBody = await request.json();
+
+    const newPaymentReminder = createPaymentReminderListItemFixture(jsonBody);
+    paymentReminderListFixture.push(newPaymentReminder);
+
+    await delay();
+    return HttpResponse.json(newPaymentReminder);
+  }),
+
+  http.post<
+    {},
+    paths['/overdue_reminders']['post']['requestBody']['content']['application/json'],
+    paths['/overdue_reminders']['post']['responses']['201']['content']['application/json']
+  >('*/overdue_reminders', async ({ request }) => {
+    const jsonBody = await request.json();
+
+    const newOverdueReminder = createOverdueReminderListItemFixture(jsonBody);
+    overdueReminderListFixture.push(newOverdueReminder);
+
+    await delay();
+    return HttpResponse.json(newOverdueReminder);
+  }),
+
+  http.patch<
+    { payment_reminder_id: string },
+    | paths['/payment_reminders/{payment_reminder_id}']['patch']['requestBody']['content']['application/json'],
+    | paths['/payment_reminders/{payment_reminder_id}']['patch']['responses']['200']['content']['application/json']
+    | paths['/payment_reminders/{payment_reminder_id}']['patch']['responses']['404']['content']['application/json']
+  >('*/payment_reminders/:payment_reminder_id', async ({ request, params }) => {
+    const jsonBody = await request.json();
+
+    const paymentReminderToUpdate = paymentReminderListFixture.find(
+      (paymentReminder) => paymentReminder.id === params.payment_reminder_id
+    );
+
+    if (!paymentReminderToUpdate) {
+      await delay();
+      return HttpResponse.json(
+        {
+          error: {
+            message: 'Payment Reminder not found',
+          },
+        },
+        {
+          status: 404,
+        }
+      );
+    }
+
+    Object.assign(paymentReminderToUpdate, jsonBody);
+
+    await delay();
+
+    return HttpResponse.json({
+      ...paymentReminderToUpdate,
+      ...jsonBody,
+    });
+  }),
+
+  http.patch<
+    { overdue_reminder_id: string },
+    paths['/overdue_reminders/{overdue_reminder_id}']['patch']['requestBody']['content']['application/json'],
+    | paths['/overdue_reminders/{overdue_reminder_id}']['patch']['responses']['200']['content']['application/json']
+    | paths['/overdue_reminders/{overdue_reminder_id}']['patch']['responses']['404']['content']['application/json']
+  >('*/overdue_reminders/:overdue_reminder_id', async ({ request, params }) => {
+    const jsonBody = await request.json();
+
+    const overdueReminderToUpdate = overdueReminderListFixture.find(
+      (overdueReminder) => overdueReminder.id === params.overdue_reminder_id
+    );
+
+    if (!overdueReminderToUpdate) {
+      await delay();
+      return HttpResponse.json(
+        {
+          error: {
+            message: 'Overdue Reminder not found',
+          },
+        },
+        {
+          status: 404,
+        }
+      );
+    }
+
+    Object.assign(overdueReminderToUpdate, jsonBody);
```

Comment:
Any specific reason to use here Object.assign intead of spread operator? {...}

Reply:
Is there a specific advantage of using the spread operator here over `Object.assign` for mocks?